src/core/Canvas.js

```diff
@@ -63,6 +63,10 @@ export class Canvas extends BaseCanvas {
         return +this._viewport.getCTM().a.toFixed(5);
     }
 
+    getConnectionBBox (connection) {
```

Comment:
Do this only work for connections ? Could we do a more general method?

Reply:
Haha mind was focused on connections :laughing: Of course this works for any element.